src/main/scala/analysis.scala

```diff
@@ -165,11 +165,24 @@ case object analysis {
       override
       val sqsInitialTimeout =
         10.hours
+
+      override
+      val terminationConfig = TerminationConfig(
+        // if true loquat will terminate after solving all initial tasks
+        terminateAfterInitialDataMappings = true,
+        // if true loquat will terminate after errorQueue will contain more
+        // unique messages than threshold
+        errorsThreshold = Some(5),
+         // maximum time for processing one task
+        taskProcessingTimeout = None,
+        // maximum time for everything
+        globalTimeout = Some(6.hours)
```

Comment:
Are you sure 6 hours will be enough for any analysis here? What if there is more data than usually?

Reply:
We had this timeout in the webapp :man_shrugging: